### ISSUE TITLE
Actions show up as extra noise in JSX panel

### ISSUE THREAD

**Issue by rodoabad** [storybookjs/addon-jsx#114]
<img width="1701" alt="Screen Shot 2020-07-09 at 10 56 54 AM" src="https://user-images.githubusercontent.com/1240667/87062632-eeaf9a00-c1d2-11ea-9298-81af9036f14f.png">

Is there anyway to remove this noise?

**Comment by hipstersmoothie** [comment]
We use a wrapper function to override the `toString` of the function


```tsx
/**
 * Overrides the toString on a function so that it addon-jsx prints
 * the callbacks in a copy-paste-able way.
 */
export const callback = <T extends Function>(fn: T): T => {
  /** A toString to render the function in storybook */
  // eslint-disable-next-line no-param-reassign
  fn.toString = () => "() => {}";
  return fn;
};
```

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v7.3.1`](https://github.com/storybookjs/addon-jsx/releases/tag/v7.3.1) :rocket:

**Comment by awdyson** [comment]
This worked for me with the v6 story writing approach.
```
export const parameters = {
  jsx: { functionValue: (fn: () => unknown) => fn.name || 'func' },
  ...
};
```
